elasticdl/python/worker/main.py

```diff
@@ -19,7 +20,10 @@ def main():
     master_channel = build_channel(args.master_addr)
 
     ps_channels = []
-    if args.ps_addrs:
+    if (
+        args.ps_addrs
```

Comment:
Why `DistributionStrategy.ALLREDUCE` still has `ps_addrs`? `ps_addrs` should set to `` for ALLREDUCE.

Reply:
Thanks. PR updated to set it to empty string instead.